libminifi/test/flow-tests/FlowControllerTests.cpp

```diff
@@ -79,6 +79,17 @@ Flow Controller:
 Remote Processing Groups:
 )";
 
+template<typename Fn>
+bool verifyWithBusyWait(std::chrono::milliseconds timeout, Fn&& fn) {
+  auto start = std::chrono::steady_clock::now();
+  while (std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now() - start) < timeout) {
```

Comment:
minor, but I don't think you need the `duration_cast`: durations in different units can be compared